espnet/asr/pytorch_backend/asr.py

```diff
@@ -976,6 +978,10 @@ def recog(args):
                             for n in range(args.nbest):
                                 nbest_hyps[n]["yseq"].extend(hyps[n]["yseq"])
                                 nbest_hyps[n]["score"] += hyps[n]["score"]
+                elif model.dmode == 'NAR':
+                    nbest_hyps = model.recognize_mask_ctc(
```

Comment:
```suggestion
                    nbest_hyps = model.recognize_nat(
```
For future research with other NAT variants, it might be better to generalize the name.